moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -363,49 +369,53 @@ export default function TheMap({
           />
         )}
         <BaseMapSourceAndLayers basemapKey={basemapKey} />
-        <ProjectSourcesAndLayers
-          isCreatingComponent={isCreatingComponent}
-          isEditingComponent={isEditingComponent}
-          isDrawing={isDrawing}
-          linkMode={linkMode}
-          clickedComponent={clickedComponent}
-          projectComponentsFeatureCollection={
-            projectComponentsFeatureCollection
-          }
-          draftEditComponent={draftEditComponent}
-        />
-        <RelatedProjectSourcesAndLayers
-          isCreatingComponent={isCreatingComponent}
-          isEditingComponent={isEditingComponent}
-          featureCollection={allRelatedComponentsFeatureCollection}
-          shouldShowRelatedProjects={shouldShowRelatedProjects}
-          clickedComponent={clickedComponent}
-        />
-        <DraftComponentSourcesAndLayers
-          draftComponentFeatures={draftComponentFeatures}
-          linkMode={linkMode}
-        />
-        <ClickedComponentSourcesAndLayers
-          clickedComponent={clickedComponent}
-          componentFeatureCollection={clickedComponentFeatureCollection}
-          isEditingComponent={isEditingComponent}
-          isClickedComponentRelated={isClickedComponentRelated}
-        />
-        <EditDraftComponentSourcesAndLayers
-          draftEditComponentFeatureCollection={
-            draftEditComponentFeatureCollection
-          }
-          linkMode={linkMode}
-          isEditingComponent={isEditingComponent}
-        />
-        <CTNSourcesAndLayers
-          isCreatingComponent={isCreatingComponent}
-          isEditingComponent={isEditingComponent}
-          isDrawing={isDrawing}
-          linkMode={linkMode}
-          ctnLinesGeojson={ctnLinesGeojson}
-          ctnPointsGeojson={ctnPointsGeojson}
-        />
+        {hasPlaceHolderLayers && (
```

Comment:
This is where we check to make sure the component placeholder layer is present before the component layers load beneath it.

Reply:
awesome that this works!